README.md

```diff
@@ -322,72 +330,6 @@ Discover our end-to-end test Dapp [here](https://e2e.ternoa.network/) to test ou
-
-The general rule of thumb is to always use the **batchAll transaction**.
-
-`batchTx` or `batchAllTx` functions also exist and return both the tx hash but not in hex format. They both work exactly the same.
-
-As an example, this can be usefull if you want to create a large collection of NFTs and Instead of creating all of them one by one, it's recommended to group them all and execute them as one transaction. This allows you to save on transaction fees and most importantly to save your time.
-
-### Let's assume we are trying to batch a mint of NFT
-In this example we are going to run the `batchAllTxHex` because we want the process to stop and revert in case of error and an hex format as result, but `batchTxHex`, `batchTx` or `batchAllTx` could be used with the exact same code below.
-
-```javascript
-export const nftsBatchMintingHex = async (nftMetadata, quantity) => {
-  // nftMetaData represents here the offChain datas of the NFT. 
-  // quantity is the number of NFT to be minted
-  try{
-    // First we create the NFT
-    const nftTx = await createNftTx(nftMetadata, 0, undefined, false)
-    // Second we add all the NFT in an Array
-    const nftsTxs = new Array(Number(quantity)).fill(nftTx)
-    // We batch the transaction with the batchAllTxHex function
-    return await batchAllTxHex(nftsTxs)
-  }catch(error){
-    console.log(error)
-  }
-}
-```
-### IPFS Upload
-IPFS is one of the solutions we recommand to upload the NFT metadatas and to provide them as the offchainData of your feature. The full IPFS Upload will be soon added to the ternoa-js SDK. For now, we invite you to look at [this](https://github.com/capsule-corp-ternoa/ternoa-js/discussions/62) github discussions or to this [Dapp](https://github.com/capsule-corp-ternoa/ternoa-workshop/blob/main/helpers/ipfs.ts) we used for a workshop before.
-
-Expected Format syntax :
-
-**Ternoa basic NFT Off-Chain Metadata - object**
-```bash
-- title: Title of the NFT - string
-- description: Description of the NFT - string
-- image: IPFS hash of the NFT's asset - string
-- properties: An object containing NFT's properties and at least a Media object - object
-	- media: An object containing NFT's asset properties - object (see below)
-
-media - object:
-- hash : IPFS hash of the NFT asset - string
-- type: Type of media (file format) - string
-- size: size of the encrypted media - string
-```
-
-**Ternoa Collection Off-Chain Metadata - object**
-
-```bash
-- banner_image: IPFS hash of the collection's banner image - string
-- name: Name of the collection - string
-- description: Description of the collection - string
-- profile_image: IPFS hash of the avatar/profile image assigned to the collection - string
-```
-
-**Ternoa Marketplace Off-Chain Metadata - object**
-
-```bash
-- name: Name of the marketplace - string
-- logo_uri: Logo URI of the marketplace - string
-```
 
 ### Connect wallet
```

Comment:
The 'Connect wallet' section can be removed.

Reply:
Done